pkg/instance/instance.go

```diff
@@ -246,6 +249,10 @@ func (ops V2DataEngineInstanceOps) InstanceDelete(req *rpc.InstanceDeleteRequest
 		}
 	}()
 
+	if req.Uuid != "" {
+		logrus.Debugf("Deleting instance %v with UUID %v. UUID is ignored in V2 data engine.", req.Name, req.Uuid)
```

Comment:
No need to mention v2 data engine is not supported in the data plane.

```suggestion
		logrus.Debugf("Deleting instance %v with UUID %v", req.Name, req.Uuid)
```